### PR TITLE
Score-client JDK update from 11 to 17

### PR DESCRIPTION
**Pull request by dahiyaAD**
Changes for score-client JDK update from 11 to 17, changing analysisId to analysis-id and studyId to study-id for the parameters-validation error message to avoid confusion. Ticket - #333

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -2,7 +2,7 @@
 # Maven builder
 ###############################
 # -alpine-slim image does not support --release flag
-FROM adoptopenjdk/openjdk11:jdk-11.0.6_10-alpine-slim as builder
+FROM adoptopenjdk/openjdk17:jdk-17-jdk-alpine as builder
```
Comment (leoraba) [inline Dockerfile:5]: adoptopenjdk project is not producing openjdk17 images.

File: Dockerfile [hunk 2]
```diff
@@ -51,20 +51,20 @@ RUN useradd $SCORE_USER  \
 # Copy client dist from previous docker build staget
 COPY --from=builder $CLIENT_DIST_DIR/ $SCORE_CLIENT_HOME
 
-# Install Open JDK 11, and remove unused things at runtime 
+# Install Open JDK 17, and remove unused things at runtime
```
Comment (leoraba) [inline Dockerfile:54]: 👍

**Review by leoraba — DISMISSED** [review]
👍

**Comment by dahiyaAD** [comment]
Making a new commit with all required changes in a single commit

**Pull request closed without merging**